package.json

```diff
@@ -20,7 +20,7 @@
   "homepage": "https://github.com/scality/S3#readme",
   "dependencies": {
     "@hapi/joi": "^17.1.0",
-    "arsenal": "git+https://github.com/scality/arsenal#7.10.51",
+    "arsenal": "git+https://github.com/scality/arsenal#62736abba4833f6aee67a66dfd320fe6317e798b",
```

Comment:
Adding another blocker for this PR as this Arsenal version is not a valid tag, let's merge it first

Reply:
updated 418144caa4a30dae305e46c66802550efd29dc34